arwen/host/arwen.go

```diff
@@ -387,6 +387,11 @@ func (host *vmHost) RunSmartContractCall(input *vmcommon.ContractCallInput) (vmO
 	return
 }
 
+// Close closes all internal instances of the vm
```

Comment:
comment does not match the behaviour

Reply:
there is nothing to close here. but it is the same for all old and new vms